### PR TITLE
Include Sinks in ToPublic Transaction

### PR DESCRIPTION
**Pull request by ferrell-code**
Description
---

- Bumps manta-rs version
- Supports new metadata changes
- Include sinks in ToPublic transaction
- fixes assetId encoding bug closes: #99 

Before we can merge this PR, please make sure that all the following items have been checked off:

- [x] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/sdk/blob/main/CHANGELOG.md) and added the appropriate `L-` label to the PR.
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.

### CODE REVIEW

File: manta-js/examples/asset-webpack-ts/index.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ const privateTransferTest = async () => {
 const toPrivateOnlySignTest = async () => {
 
     const privateWalletConfig = {
-        environment: Environment.Production,
+        environment: Environment.Development,
```
[the remark this answers is not in the document]
Comment (ferrell-code) [reply manta-js/examples/asset-webpack-ts/index.ts:111]: All of the other tests in example are set to Development except this one. I think this is correct. 

Unless we want the example pointing to testnet which in that case the other 4 need to be changed to production

[the remark this answers is not in the document]
Comment (ghzlatarev) [reply manta-js/examples/asset-webpack-ts/index.ts:111]: oh my bad i guess my other PR is not in yet. Nvm

**Review by ghzlatarev — DISMISSED** [review]

**Review by SupremoUGH — COMMENTED** [review]
The rust part is alright. I'm not approving because I think it'd be better if someone (@zqhxuyuan ?) can review the ts part more thoroughly.

**Review by zqhxuyuan — DISMISSED** [review]
LGTM

File: manta-js/examples/asset-webpack-ts/yarn.lock [hunk 2]
```diff
@@ -5435,6 +5450,11 @@ safe-buffer@5.2.1, safe-buffer@>=5.1.0, safe-buffer@^5.0.1, safe-buffer@^5.1.0,
   resolved "https://registry.npmmirror.com/safer-buffer/-/safer-buffer-2.1.2.tgz#44fa161b0187b9549dd84bb91802f9bd8385cd6a"
   integrity sha512-YZo3K82SD7Riyi0E1EQPojLz7kpepnSQI9IyPbHHg1XXXevb5dJI7tpyN2ADxGcQbHG7vcyRHk0cbwqcQriUtg==
 
+scale-codec@^0.10.0:
+  version "0.10.2"
```
Comment (zqhxuyuan) [inline manta-js/examples/asset-webpack-ts/yarn.lock:5454]: I think it's fine to bump `scale-codec` directly to `^0.10.2`.

Comment (ferrell-code) [reply manta-js/examples/asset-webpack-ts/yarn.lock:5454]: Yeah me too

File: manta-js/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased]
 
 ### Added
+[\#102](https://github.com/Manta-Network/sdk/pull/102) Include sink accounts in ToPublic
```
Comment (zqhxuyuan) [inline manta-js/CHANGELOG.md:9]: add asset id comments.

**Review by zqhxuyuan — APPROVED** [review]

**Pull request merged**